neo.UnitTests/UT_Block.cs

```diff
@@ -43,9 +43,9 @@ public void Size_Get()
         {
             UInt256 val256 = UInt256.Zero;
             TestUtils.SetupBlockWithValues(uut, val256, out var _, out var _, out var _, out var _, out var _, out var _, 0);
-            // blockbase 4 + 32 + 32 + 4 + 4 + 20 + 4
+            // blockbase 4 + 64 + 32 + 4 + 4 + 20 + 4
             // block 9 + 1
-            uut.Size.Should().Be(110);
+            uut.Size.Should().Be(114);
```

Comment:
4 + 64 + 32 + 4 + 4 + 20 + 4 = 132? 🤔  Is this test passing? haha, there is something wrong here, I believe it is the commends only (because you only increased the 4 bytes required for the long type - which is correct)

Reply:
Strange...we need to update the description